cmd/kueue/main.go

```diff
@@ -276,7 +276,10 @@ func setupControllers(ctx context.Context, mgr ctrl.Manager, cCache *cache.Cache
 	}
 
 	// setup provision admission check controller
-	if features.Enabled(features.ProvisioningACC) && provisioning.ServerSupportsProvisioningRequest(mgr) {
+	if features.Enabled(features.ProvisioningACC) {
+		if err := provisioning.ServerSupportsProvisioningRequest(mgr); err != nil {
+			setupLog.Info("Skipping provisioning controller setup: Provisioning Requests not supported (Possible cause: missing or unsupported cluster-autoscaler)")
+		}
 		// A info message is added in setupIndexes if autoscaling is not supported by the cluster
```

Comment:
The code here should go into "else" , otherwise we would invoke `provisioning.NewController` and we probably should not (as before).